dnf/conf/config.py

```diff
@@ -659,6 +661,9 @@ def __init__(self, section='main', parser=None):
         self._add_option('reposdir', ListOption(['/etc/yum.repos.d',
                                                  '/etc/yum/repos.d',
                                                  '/etc/distro.repos.d'])) # :api
+        self._add_option('modulesdir', PathOption('/etc/dnf/modules.d', abspath=True))
+        self._add_option('moduledefaultsdir',
```

Comment:
Whats the difference here? Has it sense to use them as variables? Why PathOption and not PathListOption ?